.gitmodules

```diff
@@ -0,0 +1,8 @@
+[submodule "module/linux"]
+	path = module/linux
+	url = git@github.com:utcs-scea/linux.git
```

Comment:
this is required to ignore the symlink defined below?

Reply:
No that was accidentally kept in let me fix that.